docusaurus/docs/welcome/instructions.md

```diff
@@ -49,3 +49,92 @@ npm run docs
+
+![generate-bootsplash prompt](/img/generate-bootsplash.png)
+
+* During the `generate-bootsplash` command, please provide the background color you want behind the logo during the splash screen.
+* Accept the default icon width (100dp).
+* Accept replacing the current bootsplash images.
+
+This will update the splash screen with the new logo and background color.
+
+## The `SplashScreen` component
+
+This component takes care of hiding the splash screen once the app is ready. It does, however, provide some costumizability, in the form of the following props:
+
+* **hide**: can be set to _true_ or _false_. When _false_, the splash screen will be hidden, while if _true_ it will remain desplayed. If no **hide** prop is passed (_undefined_), the splash screen will be hidden when the javascript is done loading.
+* **transitionDelay**: value in milliseconds. This value will determine the duration of the fade out transition for the splash screen.
+
+## Fullscreen image Splash Screen
+
+In order to use a predesigned fullscreen splash screen image, you will have to use a different package from the one already present in the boilerplate.
+
+Follow the steps described below:
+* Delete the `/assets/` directory from the project root.
+* Remove `bootsplash_logo.png` inside all directories matching `/android/app/src/main/res/mipmap-*dpi/`.
+* Delete the file `/android/app/src/main/res/drawable/bootsplash.xml`.
+* From inside the `/android/app/src/main/AndroidManifest.xml` file, remove the following lines:
+```xml
+<activity
+    android:name="com.zoontek.rnbootsplash.RNBootSplashActivity"
+    android:theme="@style/BootTheme"> <!-- apply the theme you created at step 3. -->
+    <intent-filter>
+        <action android:name="android.intent.action.MAIN" />
+        <category android:name="android.intent.category.LAUNCHER" />
+    </intent-filter>
+</activity>
+```
+* From inside the `/android/app/src/main/java/studio/moxy/rnwm/MainActivity.java` file, remove the following lines:
+
+```java
+import com.zoontek.rnbootsplash.RNBootSplash;
+
+...
+
+@Override
+protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    RNBootSplash.init(R.drawable.bootsplash, MainActivity.this);
+}
+```
+
+* From inside the `/ios/reactNativeWithMoxy/AppDelegate.m` file, please remove the following lines:
+
+```objective-c
+
+#import "RNBootSplash.h"
+
+[RNBootSplash initWithStoryboard:@"BootSplash" rootView:rootView];
+
+```
+
+* Delete the file `/ios/reactNativeWithMoxy/BootSplash.storyboard`.
+* Finally, uninstall the `react-native-bootsplash` package which is no longer necessary:
+```sh
+$ npm remove react-native-bootsplash
```

Comment:
`npm remove`? Shouldn't it be `npm uninstall`? 😮

Reply:
It's one of the alias for `uninstall`, maybe I should switch it to `uninstall` then 😅